drivers/flash/Kconfig

```diff
@@ -44,9 +44,14 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+choice FLASH_NRF_DRIVER
+	prompt "Nordic Semiconductor flash driver"
+  bool
+	default SOC_FLASH_NRF if !BT_LL_NRFXLIB
```

Comment:
@PavelVPV was able to get in the clock driver without mentioning BT_LL_NRFXLIB. Is it possible to use the same approach?

Reply:
I'll give it a go.

- [ ] Try to not mention BT_LL_NRFXLIB

Reply:
I see that both @PavelVPV 's driver and Zephyr's own is included in the compiled binary (I see functions from both). That should not be the case, right? It's an either-or thing?

Reply:
@thomasstenersen which functions do you see?

Reply:
(for context) I hadn't merged with the latest changes in `rugeGerritsen:develop/DRGN-10888_adapt_to_new_api`.

Now I see that the changes to kconfig makes the existing entropy and clock control driver dependent on the BT_LL_SW. I think we need to come up with some other way to achieve this.